Software/Development/SemesterProject/App/PIController.c

```diff
@@ -39,6 +38,7 @@ float PiCalculation(PIobject *PIcontroller, float reference, float measuredValue
 {
     float KP = PIcontroller->KP;
     float KI = PIcontroller->KI;
+    float KPInverse = PIcontroller->KPInverse;
 

```

Comment:
Why not just use the values inside PIcontroller instead of creating new variables? not a big deal